stock_picking_filter_lot/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock picking filter lot",
+    "summary": "In picking out lots' selection, "
+               "filter lots based on their location",
+    "version": "12.0.1.0.0",
+    "category": "Warehouse",
+    "website": "https://github.com/OCA/stock-logistics-workflow/tree/"
```

Comment:
website should be: https://github.com/OCA/stock-logistics-workflow

Reply:
I just changed it.